designer/server/src/main/scala/pl/touk/nussknacker/ui/process/deployment/DeploymentServiceImpl.scala

```diff
@@ -617,7 +617,7 @@ class DeploymentServiceImpl(
       case a if a.allowedStateStatusNames.contains(ps.status.name) => a.name
     }.distinct
     if (!allowedActionsForStatus.contains(actionName)) {
-      logger.debug(s"Action: $actionName on process: ${processDetails.name} not allowed in ${ps.status} state")
+      logger.debug(s"Action: ${actionName.value} on process: ${processDetails.name} not allowed in ${ps.status} state")
```

Comment:
We are following the approach that our value classes have overridden `toString = value` (see e.g. `ProcessName`). Thanks to that you don't need to do value in string interpolation. The good measurement for a "good value class" is the number of wrapping/unwrapping invocations - it should be minimal, especially in the production code. You can check it by find usage.

Reply:
Fixed